src/test/java/com/puppycrawl/tools/checkstyle/MainTest.java

```diff
@@ -567,4 +567,46 @@ public void testListFilesDirectoryWithNull() throws Exception {
         List<File> result = (List<File>) method.invoke(null, fileMock);
         assertEquals(0, result.size());
     }
+
+    @Test
+    public void testTranslationCheckMissinTranslationFiles() throws Exception {
```

Comment:
Why do we have this test in `MainTest`? It should belong to `TranslationCheckTest`